Description: Remove fuzz from tests/lib.rs
Forwarded: not-needed
Last-Update: 2025-10-16

This is excluded from the package as of
https://github.com/rust-lang/regex/commit/a76e0a0ef050f987d686268f1783a95b6bb25ea9.

--- a/tests/lib.rs
+++ b/tests/lib.rs
@@ -1,6 +1,5 @@
 #![cfg_attr(feature = "pattern", feature(pattern))]
 
-mod fuzz;
 mod misc;
 mod regression;
 mod regression_fuzz;
